tensorboard/components/tf_dashboard_common/tf-collapsable-pane.html

```diff
@@ -20,13 +20,11 @@
 
 <dom-module id="tf-collapsable-pane">
   <template>
-    <button
-      class="heading"
-      on-tap="togglePane"
-      open-button$="[[opened]]"
-    >
-    <span class="name">[[name]]</span>
-    <span class="count">
+    <button class="heading"
+            on-tap="togglePane"
+            open-button$="[[opened]]">
```

Comment:
Any reason to change this formatting? It's quite standard in React-land ([e.g.](https://github.com/Khan/style-guides/blob/master/style/react.md#align-and-sort-html-properties)), and I didn't write this code so it must be at least somewhat accepted in Polymer-land. :-)